.travis.yml

```diff
@@ -49,6 +53,24 @@ install:
 
   # install TensorFlow (CPU version).
   - pip install tensorflow
+  
+  # install cntk
+  - if [[ "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then
+      wget http://cntk.ai/PythonWheel/ForKeras/CPUOnly/cntk-2.0rc3-cp27-cp27mu-linux_x86_64.whl;
+      pip install cntk-2.0rc3-cp27-cp27mu-linux_x86_64.whl;
```

Comment:
Also, looks like you can upgrade from `2.0rc3` to `2.0`:
https://docs.microsoft.com/en-us/cognitive-toolkit/setup-linux-python